package.json

```diff
@@ -44,7 +44,7 @@
     "package": "ts-node ./.erb/scripts/clean.js dist && npm run build && electron-builder build --publish never",
     "rebuild": "electron-rebuild --parallel --types prod,dev,optional --module-dir release/app",
     "start": "ts-node ./.erb/scripts/check-port-in-use.js && npm run start:renderer",
-    "start:main": "cross-env NODE_ENV=development electronmon -r ts-node/register/transpile-only .",
+    "start:main": "cross-env NODE_ENV=development electron -r ts-node/register/transpile-only .",
```

Comment:
why do we want to change this?

Reply:
electronmon is a package that watch and restart elecron applications. If an application is executed using this package, it's default behaviour is when an error occurs, it will give a warning and restart the application. Even though you already catch the error, it will still restart. So in order for the main process error boundary to work, you need to use the default electron script to run the application
<img width="475" alt="Screenshot 2023-09-25 at 21 16 48" src="https://github.com/keep-starknet-strange/madara-tsukuyomi/assets/110438299/20d511b1-996c-4ad2-a48c-a166ca428c8b">

Reply:
Hmm, ok, then let's not change this but add a comment in the catch part that it only works without `electronmon`. Otherwise, without that, building becomes difficult as we need to manually restart on every change

Reply:
alright!